setup/security/authorization/index.md

```diff
@@ -92,11 +92,21 @@ hal config provider $PROVIDER account edit $ACCOUNT \
   --write-permissions role1,role2,role3
 ```
 
-(Deprecated) `requiredGroupMembership` is the old way to add access restrictions to an account. 
+(Deprecated) `requiredGroupMembership` is the old way to add access restrictions to an account.
 This method does not distinguish between `READ` and `WRITE` - users with access will have both.
 
 
 ### Applications
+Before Spinnaker 1.14, there were two types of restrictions to an application `READ` and `WRITE`.
+In the 1.14 release, a new permission type called `EXECUTE` was added. For any new applications,
+the permission required to trigger pipelines changes from groups with READ access to those with
+EXECUTE access.
+
+To maintain backwards compatibility for existing applications, groups with READ access will implicitly
+get EXECUTE access. To change this behavior for a specific application, modify the application config
+in the UI to explicitly add EXECUTE permissions to a group. You can also flip the default behavior
```

Comment:
Can you add a screen shot of the UI here?

Reply:
Oh actually, now I see there's a screenshot below. Does it want to be updated?

But I'd move the `fiat.executFallback` config sentence to after that screen.

Reply:
Added a new screenshot, and turned the two migration options into a list. The config sentence is below the screenshot